cluster/gce/util.sh

```diff
@@ -771,39 +771,78 @@ function create-nodes() {
 # - NODE_INSTANCE_PREFIX
 # - PROJECT
 # - ZONE
+# - AUTOSCALER_MAX_NODES
+# - AUTOSCALER_MIN_NODES
+function create-cluster-autoscaler-mig-config() {
+
+  # Each MIG must have at least one node.
+  if $((${AUTOSCALER_MIN_NODES} < ${NUM_MIGS})); then
```

Comment:
I think that `NUM_MIGS` refers to the number of MIGs that we have in the cluster, not the size of each of them.